src/types/wait.ts

```diff
@@ -1,7 +1,7 @@
 import {StartedGenericContainer} from "testcontainers/dist/generic-container";
 import {StartedTestContainer} from "testcontainers";
 
-export function waitForHealthCheck({command, container, expectedResult, timeoutMS = 60000, waitBeforeStart = 1000}: {
+export function waitForHealthCheck({command, container, expectedResult, timeoutMS = 120000, waitBeforeStart = 1000}: {
```

Comment:
I don't know without this, I receive a a timeout in my machine. I think this shouldn't be so slow on it, but you could check and see if the same happens to you with a lesser timeout

Reply:
That will depend on the hardware, so it's ok that for you take longer than 1 minute to startup, i think 2 minutes it's not too much time and it's ok